modules/op-running-a-pipeline.adoc

```diff
@@ -13,25 +13,31 @@ A PipelineRun starts a Pipeline and ties it to the Git and image resources that
 . Start the Pipeline for the back-end application:
 +
 ----
-$ tkn pipeline start build-and-deploy -w name=shared-workspace,claimName=source-pvc -p deployment-name=vote-api -p git-url=http://github.com/openshift-pipelines/vote-api.git -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api
+$ tkn pipeline start build-and-deploy \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
+    -p deployment-name=vote-api \
+    -p git-url=http://github.com/openshift-pipelines/vote-api.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-api \
 ----
 +
-Note the PipelineRun ID returned in the command output.
+Note that the above command uses a volume claim template, which creates a persistent volume claim for the pipeline execution.
 
-. Track the PipelineRun progress:
+. Note the `PipelineRun` ID returned in the command output and track the pipeline run progress:
 +
 ----
 $ tkn pipelinerun logs <pipelinerun ID> -f
 ----
 . Start the Pipeline for the front-end application:
 +
 ----
-$ tkn pipeline start build-and-deploy -w name=shared-workspace,claimName=source-pvc -p deployment-name=vote-api -p git-url=http://github.com/openshift-pipelines/vote-ui.git -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui
+$ tkn pipeline start build-and-deploy \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/01_pipeline/03_persistent_volume_claim.yaml \
+    -p deployment-name=vote-ui \
+    -p git-url=http://github.com/openshift-pipelines/vote-ui.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/vote-ui \
 ----
 +
-Note the PipelineRun ID returned in the command output.
-
-. Track the PipelineRun progress:
+. Note the `PipelineRun` ID returned in the command output and track the pipeline run progress:
```

Comment:
How about,

> To track the progress of the pipeline, enter the following command.
> [code block]
> where `<pipeline_id>` is ID for the pipeline that was returned by the previous command:

Or similar.

For tokens in OCP, we're using `_` underscores rather than spaces.